libs/dal/src/repositories/topic/topic.repository.ts

```diff
@@ -64,10 +64,10 @@ export class TopicRepository extends BaseRepository<EnforceEnvironmentQuery, Top
       lookup,
       topicWithSubscribersProjection,
       {
-        $limit: pagination.limit,
+        $skip: pagination.skip,
```

Comment:
In the aggregation the order matters. First we skip the amount of documents that is the resulted of the page multiplied by the page size, and then we limit the number of documents to bring all the results of that page.